### PR TITLE
One-off schema support for old graph and row plugins

### PR DESCRIPTION
**Pull request by sdboyer**
Part of #36844 

This adds one-off definitions for the old graph panel, and for row plugins.

As with everything else that we're temporarily, hackily putting in as a definition within the scuemata declaration of the overall dashboard, it's expected that:

* These will be moved out into separate files, and subsequently cuetsy'd out to typescript
* These definitions will _never_ be changed in a backwards-incompatible manner.

Maybe it makes sense that the row plugin is an exception, and should stay within the scuemata. Gotta figure it out. Either way, just emphasizing again how important is that we maintain the backwards compatibility invariant for anything included into the scuemata, but not directly declared within it. We WILL have CI to verify this.

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: cue/data/gen.cue [hunk 1]
```diff
@@ -289,6 +290,102 @@ Family: scuemata.#Family & {
                         }]
                     }
                 }
+                // Row panel
+                #RowPanel: {
+                    type: "row"
+                    collapsed: bool | *false
```
Comment (ying-jeanne) [inline cue/data/gen.cue:296]: i also changed the panel definition to type: !="" & !="graph" & !="row", in this case, i make sure all definitions have no intersection.

File: cue/data/gen.cue [hunk 2]
```diff
@@ -289,6 +290,102 @@ Family: scuemata.#Family & {
                         }]
                     }
                 }
+                // Row panel
+                #RowPanel: {
+                    type: "row"
```
[the remark this answers is not in the document]
Comment (sdboyer) [reply cue/data/gen.cue:295]: i validated this against the `elasticsearch_compare.json` dashboard, which seems to have plenty of `date_histogram` panels in it, and within rows, so i think that whatever you'd been running into isn't an issue here. ...i think? :)

File: cue/data/gen.cue (hunk 1 above)
Comment (sdboyer) [reply cue/data/gen.cue:296]: i'm confused about why this comment is anchored here

Comment (sdboyer) [reply cue/data/gen.cue:296]: i've updated over in the other spot, but only with `&& != "graph"` - i can't quite articulate why, but i'm not worried about `row` right now. maybe that's wrong :)

Comment (sdboyer) [reply cue/data/gen.cue:296]: had to revert that, it's not valid CUE. negations are hard.

unless/until we have a case that specifically shows us we need the additional exclusion of `graph` and `row` from `#Panel`, i don't think we should worry about it.

**Review by ying-jeanne — APPROVED** [review]
LGTM

**Pull request merged**